conf/test.config

```diff
@@ -96,21 +96,21 @@ profiles {
         params.three_prime_clip_r2 = 1
         params.trim_fastq          = true
     }
-    use_gatk_spark {
-        params.use_gatk_spark      = 'bqsr,markduplicates'
-    }
     umi {
         params.input               = "${baseDir}/tests/csv/3.0/fastq_umi.csv"
         params.umi_read_structure  = '7M1S+T'
     }
+    use_gatk_spark {
+        params.use_gatk_spark      = 'baserecalibrator,markduplicates'
+    }
 }
 
 //This is apparently useless as it won't overwrite things in the modules.config
 process {
```

Comment:
Think we can delete this now, no?

Reply:
Why not, we still want to test this no?

Reply:
it doesn't get overwritten but will need to be moved to the modules.config and activated somehow on test